.github/workflows/coverage.yml

```diff
@@ -120,6 +133,7 @@ jobs:
         git config --global user.email "tket-bot@cambridgequantum.com"
         git config --global user.name  "«$GITHUB_WORKFLOW» github action"
     - name: Remove old report
+      if: github.event_name == 'push'
       run: git rm -r docs/tket/test-coverage
     - name: Add report to repository
```

Comment:
Need to be a bit careful here. Looks like we are pushing both the long and non-long coverage to the same place, so they will overwrite each other. I suggest that the existing directory should be used for the long version, and a different directory used for the non-long version, so that https://cqcl.github.io/tket/tket/test-coverage/ points to the long version.

Reply:
I've changed it so that the short tests directory is `test-coverage-short`, and I've also commited the required directory and summary.txt file in th `gh-pages` branch.